content/kubermatic/master/tutorials_howtos/external_clusters/_index.en.md

```diff
@@ -5,15 +5,16 @@ weight = 7
 
 +++
 
-This section describes how to add and manage existing GKE, AKS, and EKS clusters. Once an external cluster is added and
-registered within the KKP platform, you may then upgrade the control plane version or scale the nodes.
-You can also connect any other clusters in the KKP via kubeconfig. Connected clusters can only be viewed, not edited. 
+This section describes how to add and manage existing Kubernetes clusters called as External Cluster in KKP.
+You can import or connect a cluster.
+- Import: You can import a cluster via credentials. Imported Cluster can be viewed and edited i.e, upgrade the control plane version or scale the nodes. Currently, GKE, AKS, and EKS cluster are supported.
+- Connect: You can also connect any other clusters in the KKP via kubeconfig. Connected clusters can only be viewed, not edited.
 
-The KKP platform uses existing kubeconfig or generate the new one from the cloud provider API. The KKP backend take advantage of
-this kubeconfig to retrieve the cluster information like: cluster and node details, metrics, events.
-Every cluster update is performed only by the cloud provider client. There is no need to install any agent on the cloud provider side.   
+The KKP platform uses existing kubeconfig or generates the new one from the cloud provider API.
+The KKP backend takes advantage of this kubeconfig to retrieve the cluster information like cluster and node details, metrics, events.
```

Comment:
Let's switch "cluster information like cluster and node details, metrics, events" to "cluster information, its nodes, metrics and events".